tutor/src/screens/assignment-review/publish-scores.js

```diff
@@ -3,7 +3,7 @@ import { OverlayTrigger, Tooltip } from 'react-bootstrap';
 import { AsyncButton } from 'shared';
 
 const PublishScores = observer(({ ux }) => {
-  if ((ux.scores.hasFinishedGrading && ux.scores.hasUnPublishedScore) || ux.planScores.isManualGradingGrade) { 
```

Comment:
should we keep the `ux.planScores.isManualGradingGrade`? I think we are hiding the publish button if the scores are shown after the instructor grades an answer.

Reply:
hm, yes I think that could also be removed.  Good catch!